src/commands/payid-verify.ts

```diff
@@ -0,0 +1,42 @@
+import { PaymentInformation, verifyPayId } from '../verifiable'
+import { convertJsonToVerifiedAddress } from '../verifiable/converters'
+
+import Command from './Command'
+
+/**
+ * Verifies the signatures and certs for verified addresses of the currently loaded PayID.
+ */
+export default class VerifyPayIdCommand extends Command {
+  protected async action(): Promise<void> {
+    const info = this.getPaymentInfo()
+    if (verifyPayId(info)) {
+      const addresses = info.verifiedAddresses.map((address) => {
+        return convertJsonToVerifiedAddress(address.payload)
+      })
+
+      const copy: PaymentInformation = {
+        payId: info.payId,
+        addresses,
+        verifiedAddresses: info.verifiedAddresses,
```

Comment:
Again, we're assuming `addresses === verifiedAddresses`, which is probably ok for the CLI, but might be a bad assumption.